PyPDF2/pdf.py

```diff
@@ -1843,6 +1843,35 @@ def read(self, stream):
             if line[:9] != b_("startxref"):
                 raise PdfReadError("startxref not found")
 
+        #check and eventually correct the startxref only in not strict
+        rebuildXrefTable = False
+        try:
+            stream.seek(startxref - 1,0) #-1 to check character before
+            line=stream.read(1)
+            if line not in b_("\r\n \t"):
+                raise PdfReadWarning("incorrect startxref pointer(1)",line)
```

Comment:
Warnings should not be raised, but issued with the warnings library:

```suggestion
                warnings.warn("incorrect startxref pointer(1): {}".format(line), PdfReadWarning)
```